### PR TITLE
electron-19: improve punctuation and add note

### PR DESCRIPTION
**Pull request by BlackHole1**

### CODE REVIEW

File: blog/electron-19.md [hunk 1]
```diff
@@ -1,29 +1,18 @@
...
----
+> Ref: https://github.com/electron/electronjs.org-new/blob/main/blog/electron-19.md
 
-Electron 19.0.0 已发布！ 它包括升级到 Chromium `102`, V8 `10.2`, 和 Node.js `16.14.2`。 请阅读下文了解更多详情！
+Electron 19.0.0 已发布！它包括升级到 Chromium `102`、V8 `10.2`、Node.js `16.14.2`。请阅读下文了解更多详情！
 
 ---
 
-Electron 团队很高兴发布了 Electron 19.0.0！ 您可以通过 `npm install electron@later` 进行安装，或者从我们的 [发布网站](https://www.electronjs.org/releases/stable) 下载它。 继续阅读有关此版本的详细信息，并请分享您的任何反馈！
+Electron 团队很高兴发布了 Electron 19.0.0！您可以通过 `npm install electron@later` 进行安装，或者从我们的[发布网站](https://www.electronjs.org/releases/stable) 下载它。 继续阅读有关此版本的详细信息，并请分享您的任何反馈！
```
Comment (LitoMore) [inline blog/electron-19.md]:
```suggestion
Electron 团队很高兴发布了 Electron 19.0.0！您可以通过 `npm install electron@later` 进行安装，或者从我们的[发布网站](https://www.electronjs.org/releases/stable)下载它。 继续阅读有关此版本的详细信息，并请分享您的任何反馈！
```

File: blog/electron-19.md [hunk 2]
```diff
@@ -1,29 +1,18 @@
...
 
 ## 重要变化
 
+> 从这个版本开始，安装 electron 时，可以正确解析 `HTTP(S)_PROXY` 环境变量了。即：安装 electron 时，无需配置代理软件的全局代理或者增强模式了。这一点会显著改善中国大陆使用 electron 的使用体验。
```
Comment (LitoMore) [inline blog/electron-19.md]:
```suggestion
> 从这个版本开始，安装 Electron 时，可以正确解析 `HTTP(S)_PROXY` 环境变量了。即：安装 Electron 时，无需配置代理软件的全局代理或者增强模式了。这一点会显著改善中国大陆用户使用 Electron 的体验。
```

File: blog/electron-19.md [hunk 3]
```diff
@@ -36,30 +25,31 @@ Electron 团队很高兴发布了 Electron 19.0.0！ 您可以通过 `npm instal
...
 | E15 (9月21日) | E16 (11月21日) | E17 (2月22日) | E18 (3月22日) | E19 (5月22日) |
 | ----------- | ------------ | ----------- | ----------- | ----------- |
 | 15.x.y      | 16.x.y       | 17.x.y      | 18.x.y      | 19.x.y      |
 | 14.x.y      | 15.x.y       | 16.x.y      | 17.x.y      | 18.x.y      |
 | 13.x.y      | 14.x.y       | 15.x.y      | 16.x.y      | 17.x.y      |
 | 12.x.y      | 13.x.y       | 14.x.y      | 15.x.y      | --          |
+
 ## 下一步做什么
 
 在短期内，您可以期望该团队继续专注于跟上构成 Electron 的主要组件的开发，包括 Chromium，Node 和 V8。 虽然我们谨慎地不对发布日期做出承诺，但我们的计划是大约每2个月发布一次带有新版本组件的主要版本的 Electron。
```
Comment (LitoMore) [inline blog/electron-19.md]:
```suggestion
在短期内，我们会专注于 Electron 的主要组件的开发工作，包括 Chromium，Node 和 V8。 虽然我们谨慎地不对发布日期做出承诺，但我们的计划仍是大约每两个月发布一次带有新版本组件的主要版本的 Electron。
```

**Comment by BlackHole1** [comment]
@LitoMore Done. Thanks for your review ❤️

**Review by LitoMore — APPROVED** [review]

**Pull request merged**